src/js/components/AlignmentChoiceButton.js

```diff
@@ -1,18 +1,40 @@
 import React, { Component } from 'react';
-import { saveAlignment } from '../actions/nextStepCharAction';
+import { connect } from "react-redux";
+import { bindActionCreators } from 'redux';
+import * as actionCreators from '../actions/nextStepCharAction';
+import "../../styles/attributes.scss";
 
-require("../../styles/attributes.scss");
+const AlignmentChoiceButton = ({alignment, actions, lastStep}) => {
+    let cssClass = getProperActiveElement(alignment, lastStep);
 
-export default class AlignmentChoiceButton extends Component {
-    saveAlignment() {
-        return this.props.dispatch(saveAlignment(this.props.alignment));
+    return (
+        <button class={cssClass} onClick={() => saveAlignmentToStore(alignment, actions)}>
+            <span class="attribute-choice-button__text">{alignment}</span>
+        </button>
+    );
+};
+
+const saveAlignmentToStore = (alignment, actions) => {
+    return actions.saveAlignment(alignment);
+};
+
+const getProperActiveElement = (alignment, lastStep) => {
+    if (alignment === lastStep.alignment) {
+        return 'attribute-choice-button active-element';
     }
 
-    render() {
-        return (
-            <button class="attribute-choice-button" onClick={this.saveAlignment.bind(this)}>
-                <span class="attribute-choice-button__text">{this.props.alignment}</span>
-            </button>
-        );
+    return 'attribute-choice-button';
+};
+
+const mapStateToProps = (store) => {
+    return {
+        abilities: {...store.nextStep.abilities},
```

Comment:
what 4 do u need `...` ? I think it can be just `abilities: store.nextStep.abilities`